tasks/update_go.py

```diff
@@ -24,6 +24,7 @@ def go_version(_):
         "test_version": "Whether the image is a test image or not",
         "warn": "Don't exit in case of matching error, just warn.",
         "release_note": "Whether to create a release note or not. The default behaviour is to create a release note",
+        "all_modules": "Whether to update the version in go.mod files used by otel.",
```

Comment:
I find the name not very explicit. What about a name mentioning otel, like `include_otel` so we know what this parameter does

Reply:
Agree 100% 😄 Maybe `include_otel_modules`

Reply:
I was thinking that we might have to not update some modules other than those used by otel, hence the name, but I guess there is no need to plan for something which might not happen :smile: